frontend/src/screens/apps/NewApp.tsx

```diff
@@ -180,6 +182,10 @@ const NewAppInternal = ({ capabilities }: NewAppInternalProps) => {
     }
 
     try {
+      if (apps?.some((existingApp) => existingApp.name === appName)) {
+        throw new Error("A connection with the same name already exists.");
```

Comment:
this also happens if you click edit and then save without changing, and there is no cancel button

Reply:
oops, wrong one.